ci/test_suites/authenticators_oidc/keycloak/wait_for_server

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+SERVER_LOG_PATH=$(find /opt/jboss/keycloak/standalone/log -name 'server.log.*' | head -n 1)
+
+function print_help() {
+  cat << EOF
+Wait for keycloak server to start, the script should run from inside keycloak container
+Example:
+./wait_for_server <service-name>
+EOF
+}
+
+function input_validation() {
+  local args_number="$#"
+  if [[ ${args_number} -ne 0 ]] ; then
```

Comment:
Small point: since `$#` is a well-known bash construct, 
```bash
if [[ $# != 0 ]]; then
```
should be fine (re-assignment is only needed if you want to give meaning to a numbered argument that would otherwise be opaque).  String comparison will be done by default and is equivalent to arithmetic comparison in this case, singe `$#` will always return a numeric string.

Also, the `function` keywords aren't needed.

Reply:
i _think_ that we decided to use the `function` keyword across projects

Reply:
Thanks for the comments :)
I agreed this is redundant for bash experts, but i still think this makes the script a bit clearer to understand